tests/Data/Taxonomies/TaxonomyTest.php

```diff
@@ -189,6 +189,66 @@ public function it_gets_preview_targets()
         $this->assertEquals([
             ['label' => 'Foo', 'format' => '{foo}'],
             ['label' => 'Bar', 'format' => '{bar}'],
+        ], $taxonomy->basePreviewTargets()->all());
+    }
+
+    /** @test */
+    public function it_gets_extra_preview_targets()
+    {
+        $taxonomy = (new Taxonomy)->handle('test');
+
+        $this->assertInstanceOf(\Illuminate\Support\Collection::class, $taxonomy->extraPreviewTargets());
+
+        $this->assertEquals([], $taxonomy->extraPreviewTargets()->all());
+
```

Comment:
Probably a good idea to add a base preview target as extra assurance that extraPreviewTargets doesn't include it.